static/app/views/organizationIntegrations/integrationExternalUserMappings.tsx

```diff
@@ -76,6 +78,16 @@ class IntegrationExternalUserMappings extends AsyncComponent<Props, State> {
     return externalUserMappings.sort((a, b) => parseInt(a.id, 10) - parseInt(b.id, 10));
   }
 
+  get dataEndpoint() {
+    const {organization} = this.props;
+    return `/organizations/${organization.slug}/members/`;
+  }
+
+  get baseFormEndpoint() {
```

Comment:
None of that complicated stuff above needs to be done for user mappings since the endpoint is static 🙏 😀